comfy/ldm/modules/diffusionmodules/openaimodel.py

```diff
@@ -268,7 +268,11 @@ def _forward(self, x, emb):
             if emb_out is not None:
                 if self.exchange_temb_dims:
                     emb_out = emb_out.movedim(1, 2)
-                h = h + emb_out
+                try:
+                    h = h + emb_out
+                except:
+                    emb_out = emb_out.movedim(1, 2)
+                    h = h + emb_out
```

Comment:
I think this can break compilation, and looks unsafe in general. Is it possible for you to add an explicit check?